themes/agnoster/agnoster.base.bash

```diff
@@ -129,7 +129,7 @@ bg_color() {
 # for codes (mycodes) as otherwise it'll clobber the original.
 # this changes from BASH v3 to BASH v4.
 ansi() {
-    local seq
+    local seq|
```

Comment:
This must be a typo. This will make this line and the next line (`declare -a mycodes`) run in subshells, which is equivalent to removing these two lines from the code.

Reply:
Rejected.